torchaudio/functional/functional.py

```diff
@@ -848,11 +852,11 @@ def detect_pitch_frequency(
 
 
 def sliding_window_cmn(
-    waveform: Tensor,
-    cmn_window: int = 600,
-    min_cmn_window: int = 100,
-    center: bool = False,
-    norm_vars: bool = False,
+        waveform: Tensor,
+        cmn_window: int = 600,
+        min_cmn_window: int = 100,
+        center: bool = False,
+        norm_vars: bool = False,
```

Comment:
(In general, it is a common practice not to touch the part unrelated to the main goal of the PR. You might be asked to revert it.)

Reply:
Yes, you're totally right.
I think my IDE did it when I make code reformat.